.github/workflows/maven.yml

```diff
@@ -26,4 +26,4 @@ jobs:
         distribution: 'adopt'
         cache: maven
     - name: Build with Maven
-      run: ./mvnw -B clean verify
+      run: ./mvnw -B clean install
```

Comment:
I think this is the other way around, verify phase is before install phase. We should have verify here and gpg sign in install phase.